pkg/metrics/metrics.go

```diff
@@ -298,7 +298,7 @@ func (m *ServerMetrics) SetBackupTarballSizeBytesGauge(backupSchedule string, si
 
 // SetBackupLastSuccessfull records the last time a backup ran successfully, Unix timestamp in seconds
 func (m *ServerMetrics) SetBackupLastSuccessfull(backupSchedule string) {
```

Comment:
just one `L` at the end of Successful, plus let's add Timestamp here: `SetBackupLastSuccessfulTimestamp`